shared/editor/components/Styles.ts

```diff
@@ -1935,6 +1943,98 @@ del[data-operation-index] {
     font-family: "SF Pro Text", ${props.theme.fontFamily};
   }
 }
+
+.toggle-block {
+  display: flex;
+
+  &.folded {
+    &:dir(rtl) {
+      --rotate-by: 90deg;
+    }
+    &:dir(ltr) {
+      --rotate-by: -90deg;
+    }
+    > .toggle-block-content > :is(:not(.toggle-block-head)) {
+      display: none;
+    }
+    > .toggle-block-button {
+      svg {
+        transform: rotate(var(--rotate-by));
+        pointer-events: none;
+      }
+      transition-delay: 0.1s;
+      opacity: 1;
+    }
+  }
+
+  > .toggle-block-button {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    &:has(+ .toggle-block-content > .toggle-block-head > p) {
```

Comment:
Similar, not following why these overrides are required lmk!

Reply:
These are here to set the `height` of toggle button _conditionally_ so that it's aligned as expected, based on the height of adjacent `<p>` or `<h1>-<h4>` DOM nodes. If the adjacent DOM node is `<p>`, then height of button = height of `<p>`, which in turn is a product of `line-height` and `font-size` of `<p>`. It's derived for `<h1>-<h4>` in similar manner.